laser_runner_removal/laser_runner_removal/main_node.py

```diff
@@ -219,6 +228,11 @@ def execute(self, blackboard):
     def _correct_laser(self, laser_send_point, blackboard):
         """Iteratively move the tracking laser until it is at the same point as the runner."""
         blackboard.curr_track.corrected_point = laser_send_point
+        tracking_laser = (
```

Comment:
we should centralize access to the config values

Reply:
Not really sure what you mean. Are you thinking of calling the all the config get functions a la. `tracking_laser=blackboard.main_node.get_parameter("tracking_laser").get_parameter_value().integer_array_value` in one place, and making attributes for the main node? I'm down.